components/salesflare/salesflare.app.mjs

```diff
@@ -1,11 +1,341 @@
+import { axios } from "@pipedream/platform";
+import utils from "./common/utils.mjs";
+
 export default {
   type: "app",
   app: "salesflare",
-  propDefinitions: {},
+  propDefinitions: {
+    accountIds: {
+      type: "integer[]",
+      label: "Account IDs",
+      description: "Array of account IDs",
+      optional: true,
+      async options({ prevContext }) {
+        return await utils.asyncPropHandler({
+          resourceFn: this.getAccounts,
+          page: prevContext?.page || 0,
+          labelVal: {
+            label: "name",
+            value: "id",
+          },
+        });
+      },
+    },
+    contactId: {
+      type: "integer",
+      label: "Contact ID",
+      description: "Contact ID",
+      async options({ prevContext }) {
+        return await utils.asyncPropHandler({
```

Comment:
[Is `await` really needed here](https://eslint.org/docs/latest/rules/no-return-await) and on the other async options methods below? Seems like it can be avoided